lib/utils/setPageCookies.js

```diff
@@ -0,0 +1,23 @@
+const debug = require('./debug');
+
+/**
+ * @param {!RemoteBrowser} context
+ * @param {string} url
+ */
+
+const setPageCookies = async function setPageCookies(context, url) {
+  try {
+    const { hostname } = new URL(url);
+
+    if (context.cookiesQueue.length) {
+      await context.page.setCookie(...context.cookiesQueue.map(cookie => ({
+        ...cookie,
+        domain: hostname,
+      })));
+    }
+  } catch (e) {
+    debug(e.message, 'error');
```

Comment:
Used `throw new Error` instead of `debug`. Now the error message looks like this:

```
Error: open page error: setCookie error, cookiesQueue: [{"url":"fooo","name":"fakeAuth","value":"fakeAuth"},{"url":"fooo","name":"fakeAuth1","value":"fakeAuth1"}]
```